src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsagePlugin/refresh-disk-usage.js

```diff
@@ -0,0 +1,13 @@
+function refreshDiskUsage(a, ev) {
```

Comment:
This code was broken before this PR (presumably due to some previous Jenkins core changes) so I fixed the existing code and retested. The new code works like https://github.com/jenkinsci/jenkins/pull/9787 where the callback takes two elements, the element and the event. The element isn't actually used, but it could be used to fetch data attributes for example. When the callback is executed, the POST request is made, and then we call `ev.preventDefault()` to prevent link navigation from taking place. When the POST response is received, we display a notification on the notification bar for either success or failure, using the appropriate color.

Reply:
Great stuff, thanks!